test/Unit/App/Middleware/DeprecationMiddlewareTest.php

```diff
@@ -7,9 +7,11 @@
 use Api\App\Attribute\MethodDeprecation;
 use Api\App\Attribute\ResourceDeprecation;
 use Api\App\Exception\DeprecationConflictException;
+use Api\App\Handler\HandlerTrait;
 use Api\App\Message;
 use Api\App\Middleware\DeprecationMiddleware;
```

Comment:
```suggestion
```
Double import - if you have `Subject` as an alias of `DeprecationMiddleware`, you should use only that.